lib/controllers/commit-controller.js

```diff
@@ -137,16 +139,25 @@ export default class CommitController extends React.Component {
   }
 
   @autobind
-  async commit(message) {
+  async commit(message, coAuthors) {
+
+    let msg;
     if (this.getCommitMessageEditors().length > 0) {
-      await this.props.commit({filePath: this.getCommitMessagePath()});
+      msg = this.getCommitMessageEditors()[0].getText();
     } else {
-      let formattedMessage = message;
-      if (this.props.config.get('github.automaticCommitMessageWrapping')) {
-        formattedMessage = wrapCommitMessage(formattedMessage);
-      }
-      await this.props.commit(formattedMessage);
+      const wrapMessage = this.props.config.get('github.automaticCommitMessageWrapping');
+      msg = wrapMessage ? wrapCommitMessage(message) : message;
     }
+
+    // TODO: ensure that expanded editor commit functionality still works
+    const trailers = coAuthors.map(author => {
+      return {
+        token: 'Co-Authored-By',
```

Comment:
does Atom have a style guide for user facing copy? Do we have any kind of support from a copywriter for user messaging, or do we just wing it?